core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.ts

```diff
@@ -67,6 +79,12 @@ export class OpportunitiesListItemComponent {
   opportunityButtonDisabled: boolean = false;
 
   ngOnInit(): void {
+    this.onMobile = this.windowDimensionsService.getWidth() <= 700;
```

Comment:
Please move the `700` into a constant and also add a comment explaining that it needs to be synchronized with the CSS styles (do the same in the CSS styles).

Reply:
Done